### ISSUE TITLE
The informational statement regarding account types on line 266 is too narrow

### ISSUE THREAD

**Issue by BobbyTable** [MicrosoftDocs/msteams-docs#9400]
The statement below also applies to app-registrations set up with "accounts in any organiszational directory", since this is the third option that is not listen in the box, it seems to apply to all scenarios, and hence perhaps the note could be removed? As it stands i got the impression that this section did not apply to me - but it does. It results in an auth error `CAA2000C, SubError: consent_required V2`  if not set
 
_If you've selected **Personal Microsoft accounts only** or **Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)** during app registration, update the value for the `accessTokenAcceptedVersion` property as 2._

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 52cb5b2e-e1e9-1993-04a0-6925d1453f20
* Version Independent ID: 52cb5b2e-e1e9-1993-04a0-6925d1453f20
* Content: [Register your tab app with Azure AD - Teams](https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/authentication/tab-sso-register-aad)
* Content Source: [msteams-platform/tabs/how-to/authentication/tab-sso-register-aad.md](https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/tabs/how-to/authentication/tab-sso-register-aad.md)
* Service: **msteams**
* GitHub Login: @surbhigupta
* Microsoft Alias: **surbhigupta**

**Comment by Meghana-MSFT** [comment]
Thank you for reporting this, we will check this and get back to you.

**Comment by Meghana-MSFT** [comment]
@BobbyTable - The note includes the third option as well.
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/81138249/7c02c8e5-ac26-4739-bada-9e77afdb812b)

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/81138249/7336b127-29e2-4e7a-b37f-197ab270f8fb)

Could you please confirm if this is what you mentioned?

**Comment by BobbyTable** [comment]
The note says that this applies to 2 options: The fourth and the third radio button (in that order). My registration had the second radio button selected, and i still needed to do the update.

My initial statement was a bit off since i recalled there being only three options.

**Comment by Meghana-MSFT** [comment]
@BobbyTable - We are checking this internally with team. We have the below update.

"Where is the app developer getting this error from? Do they get it back when trying to exchange the Teams-generated token for a downstream token? If so, then they need to handle that by taking the user through a consent flow or getting their admin to consent to the app.
https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/authentication/tab-sso-graph-api?tabs=dotnet#obtain-consent"

**Comment by BobbyTable** [comment]
@Meghana-MSFT No, this if from the first getAuthToken() call in MS teams, the flow ends with the error message i wrote at the top after the user consenting. If I switch the version to 2 as described in the article, the flow works. Note that im not looking for a solution to the problem (as the developers response seem to be trying to solve the problem) - I'm just suggesting that the docs be updated to also include this case

**Comment by Meghana-MSFT** [comment]
We will share the feedback internally.
